### PR TITLE
Create docs for dicom events

### PR DESCRIPTION
**Pull request by jnlycklama**
## Description
Add docs for dicom events. Based on the meatier sections of this doc https://learn.microsoft.com/en-us/azure/healthcare-apis/events/events-overview

## Related issues
Addresses [issue #].

## Testing
Describe how this change was tested.

### CODE REVIEW

**Review by mmitrik — DISMISSED** [review]
Overall this looks great, especially for a preview.  Thank you for putting it together 🙇 

I left a few specific questions inline, but no need to block this PR on them 😄

File: docs/concepts/dicomevents.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+|subject | string | Yes | The Uniform Resource Identifier (URI) of the DICOM image that was changed. Customer can access the image with the subject with https:// scheme. Customer should use the dataVersion or data.resourceVersionId to visit specific data version regarding this event.
+| eventType	| string(enum)	| Yes	| The type of change on the DICOM image.
+| eventTime	| string(datetime)	| Yes	| The UTC time when the DICOM image change was committed.
+| id	| string	| Yes	| Unique identifier for the event.
+| data	| object	| Yes	| DICOM image change event details.
+| data.imageStudyInstanceUid	| string	| Yes | The image's Study Instance UID
+| data.imageSeriesInstanceUid	| string	| Yes	| The image's Series Instance UID
+| data.imageSopInstanceUid	| string	| Yes	| The image's SOP Instance UID
+| data.serviceHostName	| string	| Yes	| The hostname of the dicom service where the change occurred. 
+| data.sequenceNumber	| int	| Yes	| The sequence number of the change in the DICOM service. Every image creation and deletion will have a unique sequence within the service.
```
[the remark this answers is not in the document]
Comment (jnlycklama) [reply docs/concepts/dicomevents.md:25]: Good call. This number can be correlated to the changefeed. I will include that in the docs

File: docs/concepts/dicomevents.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+### What DICOM image events does Events support?
+Events are generated from the following DICOM service types:
+
+DicomImageCreated - The event emitted after a DICOM image gets created successfully.
+
+DicomImageDeleted - The event emitted after a DICOM image gets deleted successfully.
+
+### What is the payload of an Events message?
+For a detailed description of the Events message structure and both required and non-required elements, see Events troubleshooting guide.
```
[the remark this answers is not in the document]
Comment (jnlycklama) [reply docs/concepts/dicomevents.md:130]: Ah yeah its another page in https://learn.microsoft.com/en-us/azure/healthcare-apis/events/events-overview. It just references the message structure. I will fix this

**Comment by wsugarman** [comment]
We don't have events as part of our OSS offering, do we? Is it misleading to have the docs here?

**Comment by mmitrik** [comment]
> We don't have events as part of our OSS offering given its integration with Event Grid, do we? Is it misleading to have the docs here?

@wsugarman, yes this is a bit misleading, but we're using this as a temporary docs location to enable a preview release.  We can remove after we have official docs published and reach GA.

**Review by mmitrik — APPROVED** [review]

**Pull request merged**